core/src/main/java/tc/oc/pgm/command/MapPoolCommand.java

```diff
@@ -43,7 +43,7 @@ public final class MapPoolCommand {
   private static final DecimalFormat SCORE_FORMAT = new DecimalFormat("00.00%");
 
   @Command(
-      aliases = {"pool", "rotation", "rot"},
```

Comment:
While these changes are going to improve voted pools overall, at the very least I think this specific change should be split off of this PR if not dropped altogether. The (non-verbal, yet still valid) feedback on #956 when this change was originally made was overwhelmingly against the change. Though I personally prefer using voted pools in my position as a map developer on OCC, there are servers in the greater PGM community that have elected to not use them and to keep static rotations. These may be pools by definition, but this does not discount the use of the word "rotation" to refer to them as dates back to OCN.

Reply:
I believe it is time to drop the outdated terms, they've been in the codebase for legacy purposes for a long while, #956 was reverted waiting for a release to be made before such a breaking change, a release was cut just yesterday so we're good to go on that department